tests/inspect/inspect.go

```diff
@@ -172,5 +176,49 @@ func main() {
 		time.Sleep(time.Duration(globalFlags.Sleep) * time.Second)
 	}
 
+	if globalFlags.Allocate >= 0 {
+		sizeBytes := globalFlags.Allocate * (1024 * 1024)
+		buf := make([]int32, sizeBytes)
+		buf[0] = 0
+
+		fmt.Println("Allocation done")
+
+		memCgroupPath, err := getOwnCgroupPath("memory")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error getting own memory cgroup path: %v\n", err)
+			os.Exit(1)
+		}
+
+		maxMemoryUsagePath := filepath.Join("/proc/1/root/", memCgroupPath, "memory.usage_in_bytes")
+
```

Comment:
Please add a comment to say that `/proc/1/root` is to escape the chroot (not to inspect memory usage of pid 1)